bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -220,6 +220,7 @@ Index of rooting profile for soil carbon
 
 Changes rooting profile from Zeng 2001 double exponential (0) to
 Jackson 1996 single exponential (1) to Koven uniform exponential (2).
+(REMOVE NOT in the CODE)
```

Comment:
```suggestion
(REMOVE, NOT in the CODE)
```
I find this comment difficult to interpret. From context I'm guessing that it means "(REMOVE, NOT in the CODE.)" If so, please add the comma between REMOVE and NOT.